pkg/controllers/dynakube/logmonitoring/reconciler.go

```diff
@@ -65,5 +68,40 @@ func (r *Reconciler) Reconcile(ctx context.Context) error {
 		return err
 	}
 
+	err = r.checkLogMonitoringSettings(ctx)
```

Comment:
Shouldn't this be a separate reconciler?
- as its rather simple this is prooobably fine as is.

You should try to limit how often we query for it, as now the logs are a bit spammy (we check it everytime)
- also we should remove the condition when logmonitoring is turned off (the setting does not need to be deleted, as that is not a requirement as of right now)

Try to do something similar that we already do for connection-info (just an example)
- Cleanup condition if necessary
- Only check the setting if condition is outdated

And if you add all this fun stuff, then having it in a separate package/reconciler makes more sense 😉

Reply:
thanks for the input. I agree totally. Will do that. 👍